### DISCUSSION TITLE
Require Closed Tickets to have a description

### DISCUSSION THREAD

**Discussion by kdecks** [RetroAchievements/RAWeb#710] [Ideas]
Disallow tickets to be closed without a description of changes made. (Similar to the requirement for users when creating tickets)

This is already an expectations for developers, building it in will help to keep developers on track and keeping good notes with less oversight.

Brief, accurate notes help verify that work was completed and help resolve future problem and times when issues resurface.

**Comment by meleu** [comment]
my ~~dream~~ long term goal is to make the tickets system work just like a github issue tracker, in other words, keeping track of everything, every change, every comment, possibility to tag other users, adding labels, etc...

**Comment by kdecks** [comment]
Asking for input in the description has all but solved this problem, but this feature would still be good.

**Comment by Jamiras** [comment]
I disagree. 

While additional input from the user does help track down the problem, it's still important for whoever closes the ticket to describe what the actual issue was and how it was fixed. That way if the fix does break something else, a history of changes can be examined to see what was changed and why. 

Ideally this information would be in the achievement's audit log (similar to git commit comments).

There are probably exceptions, like when a ticket is closed with "not enough information".

**Comment by kdecks** [comment]
Sorry, I must have not been very clear in my meaning. What I mean is
![image](https://user-images.githubusercontent.com/32706333/68093066-edadb800-fe4e-11e9-8d9c-a2f74e85cc6d.png)
Once we started asking for comments here, most devs are adding them. I also think it's important, for the same reasons you've have said.
